src/app/modals/modals.component.ts

```diff
@@ -74,7 +74,7 @@ export class ModalsComponent implements DoCheck, OnInit {
     this.modal = this.modalContainer.createComponent(factory);
     const dynamicModal = this.modal as any;
     if (data !== undefined && dynamicModal.instance.setData !== undefined) {
-      dynamicModal.instance.setData(data);
+      dynamicModal.instance.setData(data.data);
```

Comment:
Please remove this,
just remove the {data: data} -> data in modal.service

Reply:
Okay let me replace it  :)